<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addbook</class>
 <widget class="QDialog" name="addbook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>430</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>迷你简稚艺</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>添加书籍</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>430</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>迷你简稚艺</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>清空所选行</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>1151</width>
     <height>401</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="rowCount">
    <number>20</number>
   </property>
   <property name="columnCount">
    <number>7</number>
   </property>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>书名</string>
    </property>
    <property name="font">
     <font>
      <family>迷你简稚艺</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ISBN</string>
    </property>
    <property name="font">
     <font>
      <family>造字工房悦黑体验版特细体</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>数量</string>
    </property>
    <property name="font">
     <font>
      <family>迷你简稚艺</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>作者</string>
    </property>
    <property name="font">
     <font>
      <family>迷你简稚艺</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>出版社</string>
    </property>
    <property name="font">
     <font>
      <family>迷你简稚艺</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>关键字</string>
    </property>
    <property name="font">
     <font>
      <family>迷你简稚艺</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>简介</string>
    </property>
    <property name="font">
     <font>
      <family>迷你简稚艺</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>541</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>*书名，ISBN，数量，作者，出版社项为必填项</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tableWidget</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
